packages/cf-component-card/src/CardContent.js

```diff
@@ -16,7 +17,7 @@ class CardContent extends React.Component {
         {this.props.children}
         { this.props.footerMessage ? (
           <div className="cf-card__footer_message">
```

Comment:
Ah I misread your first comment as saying we _did_ need it.  Yeah, I agree, just remove the wrapper.